NPM-2.md

```diff
@@ -0,0 +1,132 @@
+Person Responsible, hereinafter referred to as the "Supplier", is obliged to
+notify Nori of any changes in land ownership and/or assignments of the right to
+act as CRC Supplier for the Registered Project in a timely manner.
+
+### Establishing the Project CO2-Equivalent Baseline
+
+As a pre-condition of Nori Project Registration, the Supplier shall establish the
+Project's Original projected 10-year CO2 Baseline by registering with and
+meeting both the land management practice and energy use reporting requirements
+of a Nori white-listed model or soil carbon stock and flux simulation tool.
+Suppliers are required to report both changes in land management practices and
+energy use, for purposes of establishing the Project baseline. Note that most
+simulation tools require Suppliers to report up to 15 years of [historical land
+management practices](NMP-???) to establish the projected 10-year project baseline.
+
+### White-Listing Requirements
+
+For a carbon stock and flux simulation tool to be
+white-listed by Nori, the owners/administrators of the tool must commit to
+the equivalent of the following data privacy standard: The simulation tool
+owner/operator/administrator will not use or share the land management and
+energy use data the Supplier provides, except for the purposes the information
+provider directly approves.  The land management and energy use data that is
+used to establish the Project 10-year Projected CO2 Baseline will not be shared
+with Nori or stored in the Nori blockchain.
+
+### White-listed Soil Carbon Stock and Flux Simulation Tools
+
+1. COMET-Farm, United States Department of Agriculture (Natural Resources
+Conservation Service) and the University of Colorado.
+
+2. ... potentially others ...
+
+### Projected Baseline Verification
+
+The Supplier shall provide the assurance of a whitelisted third party Verifier that
+the land management and energy use data supplied for use in the carbon stock and
+flux simulation tool is reasonable and accurate.
+
+### Project CO2 Baseline Quality Rating
+
+The veracity and reliability of the simulated Project 10-year CO2
+baseline will be a function of:
+
+  1. the number of years of historical land management
+     and energy use data the Supplier is able to supply
+     the simulation model administrator
+     
+  2. the accuracy of that data
+  
+  3. the current state of knowledge regarding soil quality and other factors
+     affecting the ecosystem in which the Project is located
+  
+  4. ... potentially more stuff ...
+  
+Based on guidance from Nori, along with the information provided by the Verifier,
+the administrator of the simulation tool will assign the Project Baseline to a
+"data quality tier".  The Project Baseline will be deemed one of:
+
+  1. Tier 4 -- high uncertainty
+
+  2. Tier 3 -- moderate uncertainty
+   
+  3. Tier 2 -- low uncertainty.
+   
+  4. Tier 1 -- no uncertainty.
+   
+CRCs that derive from any Registered Project will be initially listed as
```

Comment:
I would prefer that we call this a discount rate rather than listing them as "CRC1".

Reply:
I find the word "discount" very confusing. They're not being offered at a lower price...?

Reply:
It's more like an "insurance" policy. I don't want the buyers seeing the CRCs rated at different tiers, because then they'll wonder why they're getting a "lower quality" CRC than some other buyer might get. The "discount rate" means that we hold back some of the tokens the supplier has earned in an escrowed smart contract that releases them over time as verification improves.

Reply:
We are both talking about the same end result of escrow. The user doesn't really see the crc1 vs discount, and calling it a crc of a certain tier is actually more just an internal distinction and only matters to the seller (in that the person who sold the crc is only going to get nori tokens at a certain rate) and never really presented to the buyer. So discount doesn't make sense if it's only a distinction to the seller. I think this is purely a technical jargon argument to keep it clear for us in implementation rather than a ux distinction.